### DISCUSSION TITLE
Adding image to top of townhall page

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#23] [Q&A]
@brianzelip

Would I use this format  '![Covered dish flyer](../../img/covered-dish.png)' to add a image of the townhall to the top of https://github.com/Relay-Maryland/relaymaryland.com/blob/main/src%2Fpages%2Fhall.mdx? I wasn't sure if I could directly modify that file without breaking the page.

**Comment by j-p-courneya** [comment]
> @brianzelip
> 
> 
> Would I use this format  '![Covered dish flyer](../../img/covered-dish.png)' to add a image of the townhall to the top of https://github.com/Relay-Maryland/relaymaryland.com/blob/main/src%2Fpages%2Fhall.mdx? I wasn't sure if I could directly modify that file without breaking the page.

@brianzelip following up to get your feedback on this.

**Reply by brianzelip** [reply]
@j-p-courneya  Yes the format in the covered dish news post is the markdown syntax for showing an image. It's like the hyperlink syntax, but with a '!' prefix.

So adding an image that is located in /src/img/, to the Town Hall page at /src/pages/, would look something like

```md
![Description of some image](../img/some-image.png)
```

**Reply by brianzelip** [reply]
As you might guess from looking at it, .mdx files are like .md files, but with more (JavaScript) capability That's why you see the `import YoutubeVideo from '../components/YoutubeVideo.astro';` and `<YoutubeVideo id='Gx5LFxJ_JY0' title="The Relay Town Hall"/>` lines in the Town Hall page.
